include/votca/csg/molecule.h

```diff
@@ -97,7 +99,9 @@ class Molecule {
 inline Index Molecule::getBeadIdByName(const std::string &name) {
   Index i = getBeadByName(name);
   if (i < 0) {
-    { return i; }
+    {
```

Comment:
I do not understand the second layer of brackets.

Reply:
Ya, good catch, don't know how that happened.

Reply:
Might have been clang formatting...